Zombie/src/main/java/dk/gruppeseks/bodtrd/zombie/ZombieProcessor.java

```diff
@@ -21,35 +25,90 @@
 public class ZombieProcessor implements IEntityProcessor
 {
     private Map<Integer, Entity> _zombies;
-    private AISPI ai;
-    private final int MOVEMENT_SPEED = 200;
-    private final int AGGRO_RANGE = 500;
+    private AISPI _ai;
+    private final int MOVEMENT_SPEED = 130;
+    private final int AGGRO_RANGE = 500; // When it should start following a path towards the target.
+    private final int DUMB_AI_RANGE = 80; // When it shouldnt follow a path but just go directly to the target.
+    private final int TIME_BETWEEN_PATH_UPDATE = 1500;
+    private final Map<Integer, Long> zombieLastPathUpdate = new HashMap();
```

Comment:
could this not be added to the new Path class aswell? the less state we worry about in the processor the better imo